src/pages/about.js

```diff
@@ -1,66 +1,118 @@
-
-      <section className="text-center">
-        <h2 className="inline-block p-3 mb-4 text-2xl font-bold bg-yellow-400">
-          Code of Conduct
-        </h2>
+      <SEO keywords={[`nashville`, `freecodecamp`]} title="About" />
+      <section className="main-section md:bg-FCCblue-100">
+        <div className="main-section-content main-section-index-1">
+          <div className="pt-8 px-2 md:px-0 md:mr-8 md:w-8/12">
+            <h2 className=" text-FCCblue-200">
+              <span>ABOUT freeCodeCamp Nashville</span>
+            </h2>
 
-        <p className="text-left leading-loose">
-          In the interest of fostering an open and welcoming environment, we
-          pledge to making participation in our community a harassment-free
-          experience for everyone, regardless of age, body size, disability,
-          ethnicity, gender identity and expression, level of experience,
-          nationality, personal appearance, race, religion, or sexual identity
-          and orientation.
-        </p>
+            <p className="mb-5">
+              Learning to code and becoming a developer is hard work, but we’re
+              here to help. You’ll get stuck along the way. You’ll get
+              frustrated. But eventually, you’ll get it. And you’ll learn more
+              than you ever thought possible.
+            </p>
 
-        <p className="text-left leading-loose mt-4">
-          Examples of behavior that contributes to creating a positive
-          environment include:
-        </p>
+            <p className="mb-5 font-bold">
+              The skills you practice, and the connections you make here, might
+              just change your career… and your life.
+            </p>
+            <p>
+              As the Nashville chapter of the freeCodeCamp.org global learning
+              community, we are laser-focused on supporting those just starting
+              their journey in web development. We learn together and challenge
+              each other to thrive in the tech industry. We offer weekly
+              educational opportunities and code challenges centered on the
+              freeCodeCamp open source curriculum, as well as a monthly meetup,
+              networking opportunities and more.
+            </p>
+          </div>
 
-        <ul className="text-left">
-          <li>Using welcoming and inclusive language</li>
-          <li>Being respectful of differing viewpoints and experiences</li>
-          <li>Gracefully accepting constructive criticism</li>
-          <li>Focusing on what is best for the community</li>
-          <li>Showing empathy towards other community members</li>
-        </ul>
+          <div className="flex-auto pt-20 hidden md:block">
+            <LearnTeachCoding
+              className="inline-block"
+              alt="An image of Five Key words (Learn, Create, Collaborate, Teach, And Grow) suround by curry brackets"
```

Comment:
Grammar:
"An image of five key words (Learn, Create, Collaborate, Teach, and Grow) surrounded by curly brackets"

Reply:
Thank you for catching that. I will update that alt text. 👍